src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Data/blobs/input/skd/taxreport/50001337/p50001336/delegationpolicy.xml

```diff
@@ -0,0 +1,51 @@
+		<xacml:Target>
+			<xacml:AnyOf>
+				<xacml:AllOf>
+					<xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+						<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">50001336</xacml:AttributeValue>
+						<xacml:AttributeDesignator AttributeId="urn:altinn:partyid" Category="urn:oasis:names:tc:xacml:1.0:subject-category:access-subject" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
+					</xacml:Match>
+				</xacml:AllOf>
+			</xacml:AnyOf>
+			<xacml:AnyOf>
+				<xacml:AllOf>
+					<xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+						<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">SKD</xacml:AttributeValue>
+						<xacml:AttributeDesignator AttributeId="urn:altinn:org" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:resource" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
+					</xacml:Match>
+					<xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+						<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">TaxReport</xacml:AttributeValue>
+						<xacml:AttributeDesignator AttributeId="urn:altinn:app" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:resource" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
+					</xacml:Match>
+				</xacml:AllOf>
+			</xacml:AnyOf>
+			<xacml:AnyOf>
+				<xacml:AllOf>
+					<xacml:Match MatchId="urn:oasis:names:tc:xacml:1.0:function:string-equal">
+						<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#string">Sign</xacml:AttributeValue>
+						
+						<xacml:AttributeDesignator AttributeId="urn:oasis:names:tc:xacml:1.0:action:action-id" Category="urn:oasis:names:tc:xacml:3.0:attribute-category:action" DataType="http://www.w3.org/2001/XMLSchema#string" MustBePresent="false"/>
+					</xacml:Match>
+				</xacml:AllOf>
+			</xacml:AnyOf>
+		</xacml:Target>
+	</xacml:Rule>
+	<xacml:ObligationExpressions>
+		<xacml:ObligationExpression ObligationId="urn:altinn:obligation:1" FulfillOn="Permit">
+			<xacml:AttributeAssignmentExpression AttributeId="urn:altinn:obligation-assignment:1" Category="urn:altinn:minimum-authenticationlevel">
+				<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#integer">4</xacml:AttributeValue>
+			</xacml:AttributeAssignmentExpression>
+		</xacml:ObligationExpression>
+		<xacml:ObligationExpression ObligationId="urn:altinn:obligation:2" FulfillOn="Permit">
+			<xacml:AttributeAssignmentExpression AttributeId="urn:altinn:obligation-assignment:2" Category="urn:altinn:minimum-authenticationlevel-org">
+				<xacml:AttributeValue DataType="http://www.w3.org/2001/XMLSchema#integer">3</xacml:AttributeValue>
+			</xacml:AttributeAssignmentExpression>
+		</xacml:ObligationExpression>
+	</xacml:ObligationExpressions>
+</xacml:Policy>
```

Comment:
Add a line feed at the end of all files. Files should end with an empty line.

Reply:
Added empty lines at the end of all the xml files.